activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/tasks/TaskRuntimeClaimReleaseTest.java

```diff
@@ -90,17 +89,17 @@ public void cCreateStandaloneTaskReleaseUnAuthorized() {
         assertThat(standAloneTask.getStatus()).isEqualTo(Task.TaskStatus.CREATED);
 
         Throwable thrown = catchThrowable(() -> {
-                                                 // UnAuthorized release, task is not assigned
-                                                 taskRuntime.release(TaskPayloadBuilder.release().withTaskId(standAloneTask.getId()).build());
-                                             }
+                    // UnAuthorized release, task is not assigned
+                    taskRuntime.release(TaskPayloadBuilder.release().withTaskId(standAloneTask.getId()).build());
+                }
         );
         assertThat(thrown).isInstanceOf(IllegalStateException.class);
     }
 
 
-    @Test
-    @WithUserDetails(value = "garth", userDetailsServiceBeanName = "myUserDetailsService")
-    public void dCreateStandaloneTaskAndClaimAndReleaseUnAuthorized() {
+    @Test(expected = NotFoundException.class)
```

Comment:
It's better to use `catchThrowable`, as the previous test. In this way you have better control in the assertions you can do and which line is throwing the expected exception.